docs/development_suite/identity-and-access-management/user-management.md

```diff
@@ -0,0 +1,86 @@
+
+For example, it could happen that a User who does not even have permissions to access a Company from the Console, is however in a Git group with an assigned Role that gives them Git-level permissions on these Company resources!
+
+Therefore, by giving permissions on resources to Users, it is strongly suggested a double check on both Console-level permissions and Git-level ones.
+:::
+
+## Managing Company Users
+
+A User with enough administrative permission on a specific Company will be able to view the existing identities in the Company, to add new ones and to change the existing identities' Roles.
+
+<!-- TODO: aggiornare lo screenshot con la nuova versione, con filtro attivo per identity type = User -->
+![Company admin portal](../img/user-management/company_admin_portal.png)
+
+:::caution
+Please note that some permissions defined by the Company Role may be inherited on the Projects and Runtime Environment owned by the Company itself.  
+Always pay attention when assigning Roles in order to avoid providing undesired access to resources!
+:::
+
+### Adding a new User
+
+The Company Owner can add a new User by pressing the *Add user* button. The User invitation process will require an email and the Role to be assigned to the new User.
+
+<!-- TODO: SCREENSHOT OF COMPANY ADD USER MODAL -->
+
+The provided email will be searched throughout existing Users and if a match is found the User will be invited to the Company with the defined Role.
+
+:::info
+If the User does not exist in the Platform, please open a Service Request for it to be created.
+:::
+
+### Editing a User Role
+
+A User Role in the Company can be modified: to do so, simply click on the pencil-shaped button on the desired User row and select the new Role.
+
+<!-- TODO: SCREENSHOT OF COMPANY EDIT USER MODAL -->
+
+### Removing a User from the Company
+
+A User can be removed from the Company by clicking the trash icon on the table and confirming the action.
+
+<!-- TODO: SCREENSHOT OF COMPANY DELETE USER MODAL -->
+
+:::warning
+Removing a User from the Company will kick the User out of every Project and Runtime Environment. While the User may be invited back in the Company, all their previously existing Roles will be lost and cannot be recovered, meaning they must be reassigned from scratch.
+:::
+
+## Managing Project and Environment Users
+
+A User with enough administrative permission on a specific Project will be able to view all the existing identities in the Company and assign them the desired Role on the specific Project (and, optionally, on each existing Runtime Environment).
+
+![Project admin portal](../img/user-management/project_admin_portal.png)
+
+:::note
+Although the Project Identities administration portal shows all the Company's identities, this does not mean that all of the identities have access to the Project, since this depends on the Role they are assigned in the Company and how the permissions are inherited.
```

Comment:
```suggestion
Although the Project Identities administration portal shows all the Company identities, this does not mean that all of the identities have access to the Project, since this depends on the Role they are assigned in the Company and how the permissions are inherited.
```